LayoutTests/imported/w3c/web-platform-tests/html/semantics/interactive-elements/the-details-element/backdrop-descendant-selector-expected.txt

```diff
@@ -0,0 +1,21 @@
+layer at (0,0) size 800x600
```

Comment:
I'm not sure how you ended up getting Render tree dumps as baselines, but this looks definitely unexpected, especially for that many tests. I would suggest using EWS as a baseline here.